core/src/navigation/LeftNav.svelte

```diff
@@ -462,6 +462,10 @@
         : expandNavTooltip;
     }
   }
+
+  function setTitleForCategoryButton(nodes, expandedCategories){
+    return isExpanded(nodes, expandedCategories)?nodes.metaInfo.titleCollapseButton:nodes.metaInfo.titleExpandButton;
```

Comment:
(n2h) Can you make add some spacing between ```?``` and ```:``` and the text so it is readable easier? :)

Reply:
👍